catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSource.java

```diff
@@ -723,12 +735,12 @@ private GetFeatureType buildGetFeatureRequest(Query query) throws UnsupportedQue
       }
     }
     if (!queries.isEmpty()) {
-
       GetFeatureType getFeatureType = new GetFeatureType();
-      getFeatureType.setMaxFeatures(BigInteger.valueOf(query.getPageSize()));
+      getFeatureType.setMaxFeatures(maxFeatures);
```

Comment:
❓ What happens in the case on L702 above when this is null?

Reply:
Setting `maxFeatures` to null means the `maxFeatures` attribute will not be present on the outgoing hits request. Since `maxFeatures` is unspecified, the WFS server will return the total number of hits.